index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Document</title>
+  <link rel="stylesheet" href="./styles/styles.css">
+</head>
+
+<body>
+  <main id="main_viewport">
+
+    <div class="Nav-Bar">
+      <div class="box box1">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box2">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box3">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box4">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box5">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box6">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box7">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box8">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
```

Comment:
:no_good_woman:  remove these images in the navbar as the gradient has to be achieved with [CSS Gradients](https://www.w3schools.com/css/css3_gradients.asp)